ParisBNF/abb/BNFabb57.xml

```diff
@@ -103,10 +101,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
               <title type="complete" ref="LIT5798ListKingsIsrael">ነገሥት᎓ እሉ᎓ ነግሡ᎓ ለእስራኤል᎓ እሉ᎓
                 እሙንቱ᎓</title>
               <incipit xml:lang="gez"/>
-              <explicit xml:lang="gez">ቀዳሚ᎓ ኵሉ᎓ ነግሠ᎓ ሳኦል᎓ ለእስራኤል᎓ ዓመታት᎓ ፵ዘእም᎓ ነገደ᎓ ብንያም᎓ ወእምነገደ᎓
-                ይሁዳ፨ ወእምድኅረዝ᎓ ነግሠ᎓ ዳዊት᎓</explicit>
-              <incipit xml:lang="gez">ወዛካርያስ᎓ ወፈጸምያነ᎓ ትንቢት᎓ ከዊኖሙ᎓ ወዳንኤልሰ᎓ ወሕዝቅኤልሰ᎓ በፄዋዌ᎓ ተነበዩ᎓ በቢሎን᎓
-                ወኮኑ᎓ ካህናተ᎓ ኵሎሙ᎓ <hi rend="rubric">፯፡</hi>
+              <explicit xml:lang="gez">ቀዳሚ፡ ኵሉ፡ ነግሠ፡ ሳኦል፡ ለእስራኤል፡ ዓመታት፡  ፵ዘእም፡ ነገደ፡ ብንያም፡ ወእምነገደ፡ ይሁዳ፨ ወእምድኅረዝ፡ ነግሠ፡ ዳዊት፡</explicit>
+              <incipit xml:lang="gez">ወዛካርያስ፡ ወፈጸምያነ፡ ትንቢት፡ ከዊኖሙ፡ ወዳንኤልሰ፡ ወሕዝቅኤልሰ፡ በፄዋዌ፡ ተነበዩ፡ በቢሎን፡ ወኮኑ፡ ካህናተ፡ ኵሎሙ፡ <hi rend="rubric">፯፡</hi>
```

Comment:
Why did you place the incipit after the explicit? Will this be presented on the website in the same order or is the order always, that the incipit is placed before the explicit after it is transformed, as it should be?